docker/Dockerfile

```diff
@@ -3,10 +3,16 @@ FROM gcr.io/deeplearning-platform-release/pytorch-gpu.1-10
 WORKDIR /root
 
 RUN apt-get update && apt-get install -y espeak libsndfile-dev
-RUN pip install git+https://github.com/uberduck-ai/uberduck-ml-dev
+
+RUN pip install git+https://github.com/uberduck-ai/uberduck-ml-dev@zach-workflow-enhancements
```

Comment:
fix before merge?

Reply:
I have some additional changes that are in flight, so I will just keep this and continue developing in this branch